scripts/loki_transform.py

```diff
@@ -136,36 +96,33 @@ def cli(debug):
               help='Run transformation to insert custom data offload regions.')
 @click.option('--remove-openmp', is_flag=True, default=False,
               help='Removes existing OpenMP pragmas in "!$loki data" regions.')
-@click.option('--mode', '-m', default='sca',
-              type=click.Choice(['idem', 'idem-stack', 'sca', 'claw', 'scc', 'scc-hoist', 'scc-stack',
-                                 'cuf-parametrise', 'cuf-hoist', 'cuf-dynamic']),
-              help='Transformation mode, selecting which code transformations to apply.')
 @click.option('--frontend', default='fp', type=click.Choice(['fp', 'ofp', 'omni']),
               help='Frontend parser to use (default FP)')
-@click.option('--config', default=None, type=click.Path(),
-              help='Path to custom scheduler configuration file')
 @click.option('--trim-vector-sections', is_flag=True, default=False,
               help='Trim vector loops in SCC transform to exclude scalar assignments.')
 @click.option('--global-var-offload', is_flag=True, default=False,
               help="Generate offload instructions for global vars imported via 'USE' statements.")
-@click.option('--remove-derived-args/--no-remove-derived-args', default=True,
+@click.option('--remove-derived-args/--no-remove-derived-args', default=False,
               help="Remove derived-type arguments and replace with canonical arguments")
 def convert(
-        out_path, path, header, cpp, directive, include, define, omni_include, xmod,
-        data_offload, remove_openmp, mode, frontend, config, trim_vector_sections,
+        mode, config, build, source, header, cpp, directive, include, define, omni_include, xmod,
+        data_offload, remove_openmp, frontend, trim_vector_sections,
         global_var_offload, remove_derived_args
 ):
     """
-    Single Column Abstraction (SCA): Convert kernel into single-column
-    format and adjust driver to apply it over in a horizontal loop.
-
-    Optionally, this can also insert CLAW directives that may be use
-    for further downstream transformations.
+    Batch-processing mode for Fortran-to-Fortran transformations that
+    employs a :class:`Scheduler` to process large numbers of source
```

Comment:
The docstring of click commands also serves as the text served on command line as help. Therefore, I would consider this to be the only place where we should not use the Sphinx/RST link syntax with `:class:` & co.